File: Leetcoding-Actions/Explore-Learn/Arrays/heightChecker.java
class Solution {
    public int heightChecker(int[] heights) {
        /**
        * T: O(n log n) and S: O(1)
        */
        int[] s = new int[heights.length];
        for (int i = 0; i < heights.length; i ++) {
            s[i] = heights[i];
        }
        Arrays.sort(heights);
        int count = 0;
        for (int i = 0; i < s.length; i++) {
            if (s[i] != heights[i]) {
                count++;
            }
        }
        return count;
    }
}

File: Leetcoding-Actions/Explore-Learn/Arrays/heightChecker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        '''
        T: O(n log n) and S: O(1)
        '''
        count = 0
        s = sorted(heights)
        for i in range(len(s)):
            if s[i] != heights[i]:
                count += 1
        return count
